.rubocop.yml

```diff
@@ -24,6 +24,9 @@ AllCops:
 Metrics/AbcSize:
   Enabled: false
 
+Metrics/BlockLength:
+  Max: 100 # default 25
```

Comment:
Probably can disable these cops that I have bumped up? BlockLength is easily broken on simple things like....routes.rb etc. And ExampleLength/MultipleExpecations in RSpec is fine for most cases as long as it's not to a ridiculous amount.